.github/workflows/run_tests.yaml

```diff
@@ -165,7 +164,7 @@ jobs:
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
 
   push-docker-img:
-    if: github.event_name == 'push' && github.event.ref == 'refs/heads/main'
+    if: github.event.pull_request.merged == true || github.event.push.ref == 'refs/heads/main'
```

Comment:
Same as my previou comment. Is it wrong result from rebase procedure? Try to compare code with main branch, please.

https://github.com/auto-mat/ride-to-work-by-bike-frontend/blob/912c24c7b38c69f290d63f7e7e31ce40c915b11a/.github/workflows/run_tests.yaml#L166-L167

Reply:
Removed.